client/src/components/base/Navigation/index.style.ts

```diff
@@ -43,19 +81,31 @@ export const HeaderRightSection = styled.div`
   gap: 35px;
 `;
 
-export const CartWrapper = styled.div`
-  position: relative;
-`;
-
-export const BadgeWrapper = styled.div`
+export const Badge = styled.div<{ badgeContent: string }>`
   position: absolute;
   bottom: 0px;
   right: -8px;
   text-align: center;
-  line-height: 20px;
-  height: 20px;
-  width: 20px;
-  background: url(${BadgeSVG}) no-repeat center center;
+  line-height: 18px;
+  height: 18px;
+  width: 18px;
   color: #fff;
+  background-color: #2ac1bc;
```

Comment:
제 기억이 맞다면 이거 baemin-100 컬러 일텐데 아마 index.css에 선언되어 있을거예요!
```suggestion
  background-color: var(baemin-100);
```

Reply:
사실 이런 것도 Styled Component의 theme 기능을 사용하면 참 좋단 말이죠...

Reply:
수정했습니다~~~